### DISCUSSION TITLE
Enhancement: Use the interface of Method-Draw instead of Default

### DISCUSSION THREAD

**Discussion by q2apro** [SVG-Edit/svgedit#526] [Q&A]
I like the customized interface of https://github.com/duopixel/Method-Draw due to its simplicity.  Demo:  https://editor.method.ac/

However, the repository has an old version of SVGedit. It seems obsolete.

Is it possible to use the this interface with the main SVGedit distribution instead of the default one?

--

In Addition: It would be great to have a simplicity of object handling like: https://www.drawsvg.org/drawsvg.html For instance drawing an arc is extremely easy with this tool and has no learning curve. To draw an arc with SVGedit is not easy at all.

**Comment by brettz9** [comment]
Regarding your first question, you'd have to ask them. You or they are also welcome to send PRs to offer different options or themes.

Regarding your feature request, please file as a separate issue.
